website/addons/onedrive/serializer.py

```diff
@@ -0,0 +1,97 @@
+import logging
+
+from website.addons.base.serializer import OAuthAddonSerializer
+
+from website.util import api_url_for, web_url_for
+
+logger = logging.getLogger(__name__)
+
+
+class OneDriveSerializer(OAuthAddonSerializer):
+    def credentials_owner(self, user_settings=None):
```

Comment:
This should probably be using the base implementation. Currently it would throw an `AttributeError` if `user_settings` isn't passed in.